tests/e2e/script_test.go

```diff
@@ -201,7 +265,12 @@ func TestE2EScript(t *testing.T) {
 }
 
 func applyObject(h *create.Harness, obj *unstructured.Unstructured) {
-	if err := h.GetClient().Patch(h.Ctx, obj, client.Apply, client.FieldOwner("kcc-tests"), client.ForceOwnership); err != nil {
+	o := obj.DeepCopy()
+	delete(o.Object, "NO_WAIT")
+	delete(o.Object, "TEST")
```

Comment:
Good point ... any idea how was I getting away with no clearing this before?

Reply:
I think it's because of the default being `TEST: APPLY` and not having to use write it in 😆